src/Factory/AvailablePluginsJsListFactory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace RcmAdmin\Factory;
+
+use RcmAdmin\View\Helper\AvailablePluginsJsList;
+
+class AvailablePluginsJsListFactory
+{
+    public function __invoke($services)
```

Comment:
PS. This is the new signature for container-interop which is the direction ZF 2 + 3 is going